psi4/driver/procrouting/scf_proc/scf_iterator.py

```diff
@@ -964,13 +969,13 @@ def _validate_soscf():
 
 core.HF.validate_diis = _validate_diis
 
-def efp_field_fn(xyz):
+def efp_field_fn(xyz: List) -> List:
```

Comment:
Can we specify what these are lists of? It looks like the input is a list of float.